ui/page/components/proposal_list.go

```diff
@@ -192,7 +192,7 @@ func LayoutNoProposalsFound(gtx C, l *load.Load, syncing bool, category int32) D
 	}
 
 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
-	text := l.Theme.Body1(fmt.Sprintf("No %s proposals yet ", selectedCategory))
+	text := l.Theme.Body1(fmt.Sprintf("No proposals %s ", selectedCategory))
```

Comment:
This would read `No proposals under review`, `No proposals approved`, `No proposals abandoned` and `No proposals rejected`

Just to be certain are we cool with this?

Reply:
I think it's more correct than `No under review proposals yet`.  Ideally it should say "No abandoned proposals" but this would require a bigger change.

Reply:
approved by @raedah as is